test/e2e/scenario/rum/tracing.scenario.ts

```diff
@@ -28,6 +28,7 @@ describe('tracing', () => {
           })
           .then((response) => response.text())
           .then(done)
+          .catch(() => done('{}'))
```

Comment:
does that fail the test?
if it does, maybe it could be more explicit with something like:
```
.catch(() => done('{ error: "Error Reason" }'))
```

Reply:
it fails the test because the `checkRequestHeaders()` on the next line will not find the correct headers in the empty object. But a better way maybe the following:
```ts
const rawHeaders = await browserExecuteAsync<string|Error>((done) => {
  window
    .fetch(...)
    .then(...)
    .catch(() => done(new Error('Fetch request failed!')))
})
if (rawHeaders instanceof Error) {
  return fail(rawHeaders)
}
checkRequestHeaders(rawHeaders)
```
WDYT?

Reply:
lgtm